src/shortestpaths/bellman-ford.jl

```diff
@@ -37,31 +37,35 @@ function bellman_ford_shortest_paths(
     distmx::AbstractMatrix{T}=weights(graph)
     ) where T<:Real where U<:Integer
 
-    active = Set{U}(sources)
-    sizehint!(active, nv(graph))
+    active = falses(nv(graph))
+    for s in sources
+        active[s] = true
+    end
     dists = fill(typemax(T), nv(graph))
     parents = zeros(U, nv(graph))
     dists[sources] .= 0
     no_changes = false
+    new_active = falses(nv(graph))
```

Comment:
since we're using `nv(graph)` multiple times, let's memoize this with `nvg` or something.